tests/manage/pv_services/test_create_pvc_with_worker_node_down.py

```diff
@@ -0,0 +1,98 @@
+import logging
```

Comment:
I suggest moving this test into tests/manage/z_cluster/nodes so it will be executed at the end tier4a run as it involves nodes failures and more risky.
And more specifically into tests.manage.z_cluster.nodes.test_nodes_restart.TestNodesRestart as this class has some test and non-test methods that are common